CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+1.0.1
+  - GCP-43 Instance add & delete access config methods now wait for the
+  operation to finish.
```

Comment:
2 more spaces in the indent please, in this file the items are aligned to the start of the text, not to the `-`